src/main/java/net/minecraftforge/fml/common/registry/IForgeRegistry.java

```diff
@@ -40,6 +40,8 @@
 {
     public Class<V> getRegistrySuperType();
 
+    ResourceLocation getRegistryName();
+
```

Comment:
I can move it to FMLControlledNamespacedRegistry, but I thought registry names were public. Oh well.